src/main/java/com/fredsonchaves07/moviecatchapi/domain/useCases/user/CreateUserUseCase.java

```diff
@@ -97,10 +96,10 @@ private String getToken() {
         return tokenService.encrypt(Optional.of(userDTO)).getToken();
     }
 
-    private HashMap<String, Object> createMailParams(String... token) {
+    private HashMap<String, Object> createMailParams(String token) {
```

Comment:
Be more explicit. Something like creating user creation email template parameters